m1.lesson1.web_calculator/calculator.html

```diff
@@ -1,10 +1,53 @@
-<!DOCTYPE html>
-<html lang="en">
+ 
+<!DOCTYPE <!DOCTYPE html>
+<html>
 <head>
-    <meta charset="UTF-8">
-    <title>Title</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Calculator</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="mainCalc.css" />
+    <script src="mainCalc.js"></script>
 </head>
-<body>
-
+<body id="Calc" onload="main()">
+ 
+    <input id="screen" type="text"></input>
+    <!--Signs -->
+ 
+    <div>
+        <button onclick="clearScreen()" id="zero">AC</button>
+ 
+        <button onclick="plusminus()" id="plus-minus">+/-</button>
+        <button id="divide" onclick="addtoscreen(' / ')">/</button>
+ 
+    </div>
+   
+    <div>
+        <button onclick="addtoscreen(7)" id="seven" value="7">7</button>
```

Comment:
` <button onclick="addtoscreen(someValue)" value="someValue">someValue</button>`

instead, use same functions for all buttons and this.value